en/7-iterate-improve-frequently.md

```diff
@@ -32,390 +15,170 @@ altLangPage: 7-effectuer-iterations-ameliorations-constamment
-  - Having a workable solution that can be tested and validated will give you useful information for improving your service. Whenever possible, focus on results rather than unnecessary documentation and reporting (while staying within policy and regulatory limits).
-- When you can, use [agile tools and techniques](https://www.gov.uk/service-manual/agile-delivery/agile-tools-techniques). **(3. Apply agile principles and be iterative. (Do - Digital Design Playbook (ISED)))**
-  - Techniques can include: daily stand ups, issue trackers, code reviews, rapid prototyping, design sprints, usability testing, user stories, retrospective meetings.
-- Build and release a core-function oriented product that meets user needs. **(3. Apply agile principles and be iterative. (Do - Digital Design Playbook (ISED)))**
-  - Focus on developing a product that takes care of the client’s task(s). Additional capacities and new features to service can be added later as long as these add-ons do not result in the service/product losing effectiveness at core functions.
-{: .dpgn-stage-alpha .dpgn-stage-beta .dpgn-stage-live}
+- Work in an agile way
+  - Use agile tools and techniques, and continue to do so even after the service has been released 
+- follow a user-centred approach
+  - Include the user in all areas of the prototyping (design, iterations and so on)
+  - Clearly describe the lifecycle of a user story, from user research to production
+  - Ensure the team responds to user research and usability testing
+  - Explore opportunities for mobile optimization.
+- Minimum Viable Product (MVP): a product with just enough features to satisfy early customers and to provide feedback for future product development
+  - Determine the MVP
+  - Ship a functioning MVP that solves a core user need as soon as possible using a “beta” or “test” period if needed
+- Quality Assurance:
+  - Have a quality assurance testing and rollback plan that supports frequent iterations to the service
+  - Release features and improvements as soon as possible
+  - Create a prioritized list of features and bugs, also known as the “feature backlog” and “bug backlog” 
+- Teams:
+  - Keep delivery teams small and focused; limit organizational layers that separate these teams from the business owners
+  - Use a source code version control system (?) 
+  - Give the entire project team access to the issue tracker and version control system
+- Lessons Learned:
+  - Review how other organizations approach service delivery and adapt/incorporate best practices where appropriate.
+  - Explore interdepartmental and/or interjurisdictional partnership arrangements to identify opportunities for joined-up services. 
 <!-- markdownlint-enable MD032 -->
-
 </section>
 
-<section class="dpgn-section-guides">
+<section class="dpgn-section-similar">
 
-### Implementation guides
+### Other Ressources
```

Comment:
This heading is managed centrally so changing it here won't have any effect (just provided here for convenience). It can be changed to something else if there is something better (would need to consult with the other areas first though).